fe/src/main/java/org/apache/doris/load/loadv2/InsertLoadJob.java

```diff
@@ -46,14 +50,20 @@ public InsertLoadJob() {
         this.jobType = EtlJobType.INSERT;
     }
 
-    public InsertLoadJob(String label, long dbId, long tableId, long createTimestamp) {
+    public InsertLoadJob(String label, long dbId, long tableId, long createTimestamp, String failMsg) {
         super(dbId, label);
         this.tableId = tableId;
         this.createTimestamp = createTimestamp;
         this.loadStartTimestamp = createTimestamp;
         this.finishTimestamp = System.currentTimeMillis();
-        this.state = JobState.FINISHED;
-        this.progress = 100;
+        if (Strings.isNullOrEmpty(failMsg)) {
+            this.state = JobState.FINISHED;
+            this.progress = 100;
+        } else {
+            this.state = JobState.CANCELLED;
+            this.failMsg = new FailMsg(CancelType.ETL_RUN_FAIL, failMsg);
```

Comment:
There is no ETL_RUN_FAIL in streaming load. Please use LOAD_RUN_FAIL instead.

Reply:
OK